test/mpi/ft/testlist

```diff
@@ -20,3 +20,4 @@ agree_shrink 4 env=MPIR_CVAR_ENABLE_FT=1 mpiexecarg=-disable-auto-cleanup result
 revoke_shrink 4 env=MPIR_CVAR_ENABLE_FT=1 mpiexecarg=-disable-auto-cleanup resultTest=TestStatusNoErrors strict=false timeLimit=10 xfail=ticket2198
 nbccoll 4 env=MPIR_CVAR_ENABLE_FT=1 mpiexecarg=-disable-auto-cleanup resultTest=TestStatusNoErrors strict=false timeLimit=10 xfail=ticket2237
 failure_ack 3 env=MPIR_CVAR_ENABLE_FT=1 mpiexecarg=-disable-auto-cleanup resultTest=TestStatusNoErrors strict=false timeLimit=10 xfail=ticket2237
+subcomm 4 mpiexecarg=-disable-auto-cleanup timeLimit=10 strict=false resultTest=TestStatus
```

Comment:
Should this test go in the `errors` directory instead of the `ft` directory? This isn't exactly FT and doesn't require the FT runtime flags to be set I think.

Reply:
Good point. Let me change it.